### PR TITLE
fix: Add Create Portal to Tooltip component

### PR DESCRIPTION
**Pull request by kbesingeryeh**
Added a fixed strategy so the tooltip component will always sit on top of anything. 

Screen Shot 2021-08-23 at 3 03 37 PM

### CODE REVIEW

**Review by ksk5280 — COMMENTED** [review]
@kbesingeryeh I think you might want to change the commit title to `fix` instead of `style` because I don't think `style` will trigger a release.

**Comment by stephenh** [comment]
@bdow @KoltonG @kbesingeryeh / all I saw this impressively detailed post go by just a few days ago, which is also about popper coincidentally, and that attaching it to a dummy div via a portal would be a good strategy:

https://atfzl.com/don-t-attach-tooltips-to-document-body

If we did that with a `z-index` that puts it above the main content, would that also fix this issue?

Not really sure the pros/cons of each...

File: src/forms/BoundToggleChipGroupField.tsx [hunk 1]
```diff
@@ -4,10 +4,7 @@ import { ToggleChipGroup, ToggleChipGroupProps } from "src/inputs";
 import { useTestIds } from "src/utils";
 import { defaultLabel } from "src/utils/defaultLabel";
 
-export type BoundToggleChipGroupFieldProps = Omit<
-  ToggleChipGroupProps,
-  "values" | "onChange" | "label"
-> & {
+export type BoundToggleChipGroupFieldProps = Omit<ToggleChipGroupProps, "values" | "onChange" | "label"> & {
```
Comment (kbesingeryeh) [inline src/forms/BoundToggleChipGroupField.tsx:7]: This is a prettier change when I ran yarn format

File: src/forms/BoundToggleChipGroupField.tsx [hunk 2]
```diff
@@ -20,15 +17,7 @@ export function BoundToggleChipGroupField(props: BoundToggleChipGroupFieldProps)
...
-      {() => (
-        <ToggleChipGroup
-          label={label}
-          values={field.value || []}
-          onChange={onChange}
-          {...testId}
-          {...others}
-        />
-      )}
+      {() => <ToggleChipGroup label={label} values={field.value || []} onChange={onChange} {...testId} {...others} />}
```
Comment (kbesingeryeh) [inline src/forms/BoundToggleChipGroupField.tsx:20]: This is a prettier change when I ran yarn format

File: src/components/Tooltip.tsx [hunk 3]
```diff
@@ -55,22 +56,19 @@ function Popper({ triggerRef, content, placement = "auto" }: PopperProps) {
   const [arrowRef, setArrowRef] = useState<HTMLDivElement | null>(null);
 
   const { styles, attributes } = usePopper(triggerRef.current, popperRef.current, {
+    strategy: "fixed",
```
Comment (bdow) [inline src/components/Tooltip.tsx:59]: Will this cause the tooltip to stay in place when the user scrolls the page, or scrolls the trigger out of the viewport? I know the tooltip will disappear soon after the user is no longer hovering over the trigger, but just curious if the `fixed` strategy will prevent the Tooltip staying next to the trigger?

File: src/components/Modal/useModal.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import { useContext, useMemo, useRef } from "react";
 import { BeamContext } from "src/components/BeamContext";
 import { Callback, CheckFn } from "src/types";
+import { maybeCall } from "src/utils";
 import { ModalProps } from "./Modal";
-import {maybeCall} from "src/utils";
```
Comment (kbesingeryeh) [inline src/components/Modal/useModal.tsx:5]: This is a prettier change when I ran yarn format

File: README.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ All of these points are generally in stark contrast to traditional, "big" UI too
 
 For them, a MUI application in Company A shouldn't have to look & behave exactly like a MUI application in Company B. Which makes sense.
 
-But for Beam at Homebound, we specifically *want* a component that behaves in our App A to look & behave the same as it does in our App B.
+But for Beam at Homebound, we specifically _want_ a component that behaves in our App A to look & behave the same as it does in our App B.
```
Comment (kbesingeryeh) [inline README.md:44]: This is a prettier change when I ran yarn format

File: src/components/Tooltip.tsx (hunk 3 above)
Comment (kbesingeryeh) [reply src/components/Tooltip.tsx]: We removed the fixed strategy and it still works

**Review by bdow — APPROVED** [review]
Good quick fix! Thanks!

**Review by KoltonG — APPROVED** [review]
🍣

**Pull request merged**